src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -128,6 +130,7 @@ static void print_help_messages() {
     message_list.emplace_back("OV_GPU_Help", "Print help messages");
     message_list.emplace_back("OV_GPU_Verbose", "Verbose execution");
     message_list.emplace_back("OV_GPU_VerboseColor", "Print verbose color");
+    message_list.emplace_back("OV_GPU_VerboseFile", "Filename to dump verbose log");
```

Comment:
Why do we need that? You can just redirect cerr and cout to the file if you want, i.e.
```
$ OV_VERBOSE=4 ./my_app &> log.txt
```

Reply:
When application is launched from cmdline, it is possible. But in some cases, it is launched from windows application directly. I think that is why clintercept has similar option.